content/docs/start/experiments-trail/experiments.md

```diff
@@ -0,0 +1,319 @@
+---
+title: 'Get Started: Experiments'
+---
+
+# Get Started with Experiments
+
+<abbr>Experiments</abbr> proliferate quickly in ML projects where there are many
+parameters to tune or other permutations of the code. We can organize such
+projects and keep only what we ultimately need with `dvc experiments`. DVC can
+track experiments for you so there's no need to commit each one to Git. This way
+your repo doesn't become polluted with all of them. You can discard experiments
+once they're no longer needed.
+
+Previously, we learned how to tune [ML pipelines](/doc/start/data-pipelines) and
+[compare the changes](/doc/start/metrics-parameters-plots). In this section, we
+will explore the basic features of DVC experiment management with
+[`get-started-experiments`][gse] project.
+
+[gse]: https://github.com/iterative/get-started-experiments
+
+## ⚙️ Installing the Example Project
+
+These commands are run in the [`get-started-experiments`][gse] project. You can
+run the commands in this document after cloning the repository and installing
+the requirements.
+
+### 👥 Clone the project and create virtual environment
```

Comment:
emojis: I think we should use the basic style that we already have (less emojis), it's a bit too playful now :)

Reply:
Too playful, you say :))

In the first version I didn't put any and it looked a bit grim to me. The current emoji set doesn't correspond to these topics. I can put a "gear" here but gear here, gear there, it becomes too gearful. :)

Let me look at this.